From c881365edb49a1944d06ceed94a531340cf26a9b Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Tue, 5 Oct 2021 20:05:42 -0700
Subject: [PATCH 09/15] SystemUI: Remove Android build number from QS footer

This is only shown when developer settings is enabled, but we developers
tend to always have it enabled. Irrelevant text showing up in the QS
footer is a bit ugly and it's not really useful when this information is
easily accessible in Settings anyway.

Change-Id: Ideb38eeb603dc5c51945fafeae9c4eca18531b5d
---
 .../com/android/systemui/qs/QSFooterView.java | 19 ++++---------------
 1 file changed, 4 insertions(+), 15 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/qs/QSFooterView.java b/packages/SystemUI/src/com/android/systemui/qs/QSFooterView.java
index 61905ae00..107e65c25 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/QSFooterView.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/QSFooterView.java
@@ -85,20 +85,9 @@ public class QSFooterView extends FrameLayout {
 
     private void setBuildText() {
         if (mBuildText == null) return;
-        if (DevelopmentSettingsEnabler.isDevelopmentSettingsEnabled(mContext)) {
-            mBuildText.setText(mContext.getString(
-                    com.android.internal.R.string.bugreport_status,
-                    Build.VERSION.RELEASE_OR_CODENAME,
-                    Build.ID));
-            // Set as selected for marquee before its made visible, then it won't be announced when
-            // it's made visible.
-            mBuildText.setSelected(true);
-            mShouldShowBuildText = true;
-        } else {
-            mBuildText.setText(null);
-            mShouldShowBuildText = false;
-            mBuildText.setSelected(false);
-        }
+        mBuildText.setText(null);
+        mShouldShowBuildText = false;
+        mBuildText.setSelected(false);
     }
 
     @Override
@@ -188,4 +177,4 @@ public class QSFooterView extends FrameLayout {
     private void updateVisibilities() {
         mBuildText.setVisibility(mExpanded && mShouldShowBuildText ? View.VISIBLE : View.INVISIBLE);
     }
-}
\ No newline at end of file
+}
-- 
2.42.1

